torchrl/objectives/ppo.py

```diff
@@ -363,7 +361,7 @@ def critic(self):
 
     @property
     def actor_params(self):
-        logging.warning(
+        warnings.warning(
```

Comment:
```suggestion
        warnings.warn(
```

I think the syntax is this